templates/email/order-cancel.hypr

```diff
@@ -49,6 +49,8 @@
             </thead>
             {% for item in model.items %}
                 <tbody class="mz-ordersummary-lineitems">
+                {% if ((not item.originalLineId) && model.substitutedItems.length > 0) %}
```

Comment:
@kibo-snehanerekar I see the following conditional statement across most of the templates in your PR and I'm wondering if the check should include an OR for cases where there are no substitutions on the shipment. For example, OR `(not model.substitutedItems.length > 0)` ...maybe I'm reading it wrong.
```
{% if ((not item.originalLineId) && model.substitutedItems.length > 0) %}
```